mlir/lib/Dialect/SparseTensor/Transforms/SparseGPUCodegen.cpp

```diff
@@ -1269,7 +1269,7 @@ struct LinalgOpRewriter : public OpRewritePattern<linalg::GenericOp> {
     AffineExpr i, j, k;
     bindDims(getContext(), i, j, k);
 
-    // TODO: more robust patterns, tranposed versions, more kernels,
+    // TODO: more robust patterns, transposed versions, more kernels,
```

Comment:
When I saw the title, I just knew it would also involve a sparse file, since I always seem to make this typo ;-)

LGTM